python/hail/linalg/blockmatrix.py

```diff
@@ -288,23 +378,90 @@ def filter(self, rows_to_keep, cols_to_keep):
         return BlockMatrix(self._jbm.filter(jarray(Env.jvm().long, rows_to_keep),
                                             jarray(Env.jvm().long, cols_to_keep)))
 
+    @typecheck_method(uri=str)
+    def tofile(self, uri):
+        """Collects and writes data to a binary file.
+
+        Examples
+        --------
+
+        >>> from hail.linalg import BlockMatrix
+        >>> import numpy as np
+        >>> bm = BlockMatrix.random(10, 20)
+        >>> bm.tofile('file:///local/file') # doctest: +SKIP
+
+        To create a :class:`numpy.ndarray` of the same dimensions:
+
+        >>> a = np.fromfile('/local/file').reshape((10, 20)) # doctest: +SKIP
+
+        See also :meth:`.to_numpy`.
+
+        Notes
+        -----
+        The number of entries must be less than :math:`2^{31}`.
```

Comment:
This comment is unnecessary, it already is.

Reply:
? Both a BlockMatrix and a NumPy array (in 64bit Python) can have more than 2^31 entries.

Reply:
Ah, because you go through breeze.  Nevermind.